infer/environment.cc

```diff
@@ -1469,6 +1469,10 @@ core::TypePtr Environment::processBinding(core::Context ctx, const cfg::CFG &inW
                             e.setHeader("`{}` is useless because `{}` is already a subtype of `{}`", "T.cast",
                                         ty.type.show(ctx), castType.show(ctx));
                             e.addErrorSection(ty.explainGot(ctx, ownerLoc));
+                            auto argLoc = core::Loc{ctx.file, c.valueLoc};
```

Comment:
if you want you could use locAt here too

Reply:
Gah, I can't even remember to use helpers that I added in the first place.